### DISCUSSION TITLE
idea for a new dextensify

### DISCUSSION THREAD

**Discussion by SIMONHSCH** [3kh0/ext-remover#1024] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I think you can improve Dextensify by refreshing iframes instead of create a lot of iframes beacause it makes the chromebook very slow.

**Comment by AshtonDavies** [comment]
> I think you can improve Dextensify by refreshing iframes instead of create a lot of iframes beacause it makes the chromebook very slow.

Webpages do not have the ability to refresh secure iframes, which is why Dextensify creates a lot of iframes and then slows it down a bit to let Chrome garbage collect them. If Chrome does not garbage collect when it is expected to, your Chromebook can crash or freeze up too much.

Read the [documentation](https://ading.dev/blog/posts/dextensify.html) for details on this.

**Reply by SIMONHSCH** [reply]
We can also try to set how many iframes would be created.

**Reply by stuartcatt** [reply]
would something like this work?
`var i = 1
while (i > 1000 ) {
var x = document .createElement("iframe")
x.src = "https://www.example.com"
// I dont know what the url in dextensify is
setTimeout(x.remove, 200);
document.body.appendChild(x)
i = i + 1
};`

**Reply by SIMONHSCH** [reply]
where do I need to insert that in the Dextensify program 😅

**Reply by Aka-but-nice** [reply]
If you want your device in a frozen state, then of course, be my guest.

**Reply by SIMONHSCH** [reply]
So, what are some ways to freeze extensions other than LTmeat print and LTmeat flood ?

**Comment by Blobby-Boi** [comment]
> I think you can improve Dextensify by refreshing iframes instead of create a lot of iframes beacause it makes the chromebook very slow.

Lol this is what I did for the exthanger thing but it doesn’t work that well

**Reply by Blobby-Boi** [reply]
You can’t refresh the iframes but you can set the src to blank and then back
